src/api/models/profiles/hideProfile.ts

```diff
@@ -0,0 +1,25 @@
+import { Response } from 'express';
+import mongoose from 'mongoose';
+import { IProfileDoc } from '../../../mongodb/profiles.db';
+import { updateProfile } from './updateProfile';
+
+const model = mongoose.model<IProfileDoc>('Profiles');
```

Comment:
We do not need to define a `model` variable since it is not used in this file. Let's remove that line.